WordPress/openverse-frontend
# Remove `nuxt-ssr-cache` and move dependencies

## Fixes


Fixes #414 by @zackkrida
Related to #388 by @rbadillap 

## Description


Doing the following changes should prepare for the deployment process with Docker.
- Removes `nuxt-ssr-cache`and `patch-package`
- Removes the postinstall script and the patches folder
- Moves `sass-loader` and `sass` to devDependencies
- Moves `core-js`, `glob` and `vue-i18n-extract` to production dependencies

## Testing Instructions

1. Run `npm run install && npm run dev`
2. Check the site at `http://localhost:8443/`
3. ...

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
